### PR TITLE
chore: add new frontend resources

### PR DESCRIPTION
**Pull request by darkweb907**
updating site error

### CODE REVIEW

**Review by CBID2 — APPROVED** [review]
LGTM! 👍🏾 😄

**Review by rupali-codes — COMMENTED** [review]
Hey, why have you removed the resources from design-inspiration.json?

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @darkweb907! 😄 I provided a suggestion to improve the description. I hope you find it helpful! 😄

**Comment by CBID2** [comment]
Hi @darkweb907. Fix the branch conflicts please.

**Review by rupali-codes — CHANGES_REQUESTED** [review]
You have resolved conflicts incorrectly, please checkout the file carefully. Let' us know if you need any help

**Comment by rupali-codes** [comment]
@darkweb907 are you still working on this?

**Comment by darkweb907** [comment]
> @darkweb907 are you still working on this?

Sorry will get back to it soon but currently writing exam now

**Comment by rupali-codes** [comment]
> > @darkweb907 are you still working on this?
> 
> Sorry will get back to it soon but currently writing exam now

Okay, good luck!

**Comment by rupali-codes** [comment]
@darkweb907 hey, are you done with your exams?

**Comment by darkweb907** [comment]
Yea

**Comment by darkweb907** [comment]
Yes, I am!
Done check it

**Comment by darkweb907** [comment]
Check it now

**Review by CBID2 — DISMISSED** [review]
LGTM! :)

**Comment by rupali-codes** [comment]
@darkweb907 hey your vercel deployment is getting failed bcz of the following issue

image

**Comment by rupali-codes** [comment]
@darkweb907 does that PR resolves any issue? if so, please reference that here

File: database/frontend/design-inspirations.json [hunk 1]
```diff
@@ -125,13 +126,22 @@
...
     },
     {
+
+        "name": "Uiverse",
+        "description": "this is a completely free website where you can build cool solution with HTML and CSS. there's is also lots of CSS challenges to show off your skills",
+        "url": "https://uiverse.io",
+        "category": "frontend"
+        "subcategory": "design-inspiration"
+      },
+
```
Comment (rupali-codes) [inline database/frontend/design-inspirations.json:136]:
```suggestion
```

File: database/frontend/design-inspirations.json [hunk 2]
```diff
@@ -163,6 +173,7 @@
         "name": "SaasInterface",
         "description": "Save yourself hours of research and create beautiful apps by taking inspiration from the best SaaS products.",
         "url": "https://saasinterface.com/",
+
```
Comment (rupali-codes) [inline database/frontend/design-inspirations.json:176]:
```suggestion
```

File: database/frontend/design-inspirations.json [hunk 3]
```diff
@@ -1,5 +1,5 @@
 [ {
-        "name": "Dribbble",
+        "name": "Dribble" ,
```
Comment (rupali-codes) [inline database/frontend/design-inspirations.json:2]:
```suggestion
        "name": "Dribble",
```

**Review by rupali-codes — CHANGES_REQUESTED** [review]

File: database/frontend/design-inspirations.json (hunk 3 above)
Comment (darkweb907) [reply database/frontend/design-inspirations.json]: Resolve

**Comment by CBID2** [comment]
@darkweb907, you have some merge conflicts that need to be resolved.

**Comment by CBID2** [comment]
I closed this pull request since it's becoming stale. When you're ready to work on it again @darkweb907, let us know. 😄  👍🏾

**Pull request closed without merging**